parallel-consumer-core/src/test/java/io/confluent/parallelconsumer/offsets/WorkManagerOffsetMapCodecManagerTest.java

```diff
@@ -359,6 +360,45 @@ void deserialiseBitSet() {
         assertThat(deserialisedBitSet).isEqualTo(input);
     }
 
+    @SneakyThrows
+    @Test
+    void deserialiseKafkaStreamsV1() {
+        final var input = ByteBuffer.allocate(32);
+        // magic number
+        input.put((byte) 1);
+        // timestamp
+        input.putLong(System.currentTimeMillis());
+
+        EncodedOffsetPair encodedOffsetPair = EncodedOffsetPair.unwrap(input.array());
+        assertThatThrownBy(()->encodedOffsetPair.getDecodedIncompletes(0L))
+                .isInstanceOf(KafkaStreamsEncodingNotSupported.class)
+                .hasMessage("It looks like you're reusing a Kafka Streams consumer group id. This isn't supported. Please, use a fresh consumer group, unique to PC");
```

Comment:
just check for keys words in the message, not the whole sentence. makes the test less brittle.